### PR TITLE
cleanup groups at the start of the tests

### PR DESCRIPTION
**Pull request by mwrock**

### CODE REVIEW

**Comment by rhass** [comment]
Is there any way we can set the user id range in a higher range such as >= 30000?

**Comment by rhass** [comment]
:+1:

File: spec/functional/resource/group_spec.rb [hunk 1]
```diff
@@ -169,8 +173,11 @@ def create_group
 
     describe "when the users exist" do
       before do
+        high_uid = 60000
```
Comment (lamont-granquist) [inline spec/functional/resource/group_spec.rb]: little worried that could interact badly/surprisingly with systems that have 32-bit signed ints for uids.   i can't recall when linux switched.  no idea when/if solaris or aix is off of those.  dunno...

Comment (mwrock) [inline spec/functional/resource/group_spec.rb]: It passes all of our test nodes. I ran an ad hoc build on this.

Comment (lamont-granquist) [inline spec/functional/resource/group_spec.rb]: make that 16-bit signed ints...   i can't even think that small any more...

Comment (mwrock) [inline spec/functional/resource/group_spec.rb]: ok i think thats fine. knocked it down to 30000

**Pull request merged**